src/kernel/patch/om-box.lisp

```diff
@@ -47,7 +47,7 @@
     (text-align :initform nil :accessor text-align :initarg :text-align)
     (icon-pos :initform :left :accessor icon-pos :initarg :icon-pos)
     (show-name :initform t :accessor show-name :initarg :show-name)
-    (show-markers :accessor show-markers :initform t)
+    (show-markers  :initform t :accessor show-markers :initarg :show-markers)
```

Comment:
Can you remove this little extra-space :neckbeard:  ?

Reply:
done. I think.

Reply:
Is there a particular code formatter I/we should be using?

Reply:
👍 
However, talking about details / process you should in this case rebase your branch and fix this up in the original commit, rather than fixing it by another commit.

Reply:
> Is there a particular code formatter I/we should be using?

Good question. I was looking into this recently. We should probably consider that, but I don't know if there's good ones for Common Lisp...